custom_typings/custom.d.ts

```diff
@@ -0,0 +1,6 @@
+declare var PR;
```

Comment:
this tells typescript that there is a global PR variable which we use [here](https://github.com/aurelia-ui-toolkits/carmel-fe/blob/master/src/shared/markdown.ts#L25)